iothub_client/inc/iothub_message.h

```diff
@@ -42,6 +42,12 @@ IOTHUBMESSAGE_UNKNOWN \
 */
 MU_DEFINE_ENUM(IOTHUBMESSAGE_CONTENT_TYPE, IOTHUBMESSAGE_CONTENT_TYPE_VALUES);
 
+#define IOTHUB_MESSAGE_CATEGORY_VALUE   \
+    MESSAGE_CATEGORY_TELEMETRY,         \
+    MESSAGE_CATEGORY_SECURITY               // Note: Security messages are special messages only accessable through the ASC portal
+
+MU_DEFINE_ENUM(IOTHUB_MESSAGE_CATEGORY, IOTHUB_MESSAGE_CATEGORY_VALUE);
```

Comment:
>IOTHUB_MESSAGE_CATEGORY [](start = 15, length = 23)

I still think these are "routes" not "categories". I think it would be safe to name this as "IOTHUBMESSAGE_ROUTE" and have "MESSAGE_ROUTE_EVENTHUB" and "MESSAGE_ROUTE_SECURITY". If the service team documented the system properties as "routes", that would help us close on this quicker.